src/app/withdraw/[txid]/components/withdrawal-status.tsx

```diff
@@ -113,20 +136,31 @@ function Content(initialData: Props) {
         <WithdrawalStepper status={status} txId={txid} />
       </div>
 
-      {txid && (
-        <div className="w-full flex-row flex justify-between items-center">
+      <div className="w-full flex-row flex justify-between items-center">
+        {stacksTx && (
           <a
             className="w-40 rounded-lg py-3 flex justify-center items-center flex-row bg-orange"
-            href={`https://explorer.hiro.so/txid/${txid}?chain=${
-              bridgeConfig.WALLET_NETWORK === "mainnet" ? "mainnet" : "testnet"
-            }`}
+            href={getExplorerUrl(
+              stacksTx,
+              getStacksNetwork(bridgeConfig.WALLET_NETWORK),
+            )}
             target="_blank"
             rel="noreferrer"
           >
             View stacks tx
           </a>
-        </div>
-      )}
+        )}
```

Comment:
We should probably keep a button to link to the submitted tx (the init withdrawal) -- either a separate one, or reusing this one if `stacksTx` is not set. This current one, coming from `fulfillment`, is the stacks txid of the withdrawal accept/reject, so you only have it after the request is completed.

Reply:
fair I can keep it but the problem here is that i need it from the emily api after the switch to fetching the request from emily

Reply:
and also we might need the reject tx, basically accepted withdrawals get a special treatment